sdk/neonpostgres/Azure.ResourceManager.NeonPostgres/api/Azure.ResourceManager.NeonPostgres.netstandard2.0.cs

```diff
@@ -0,0 +1,254 @@
+        protected MockableNeonPostgresResourceGroupResource() { }
+        public virtual Azure.Response<Azure.ResourceManager.NeonPostgres.OrganizationResource> GetOrganizationResource(string organizationName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.ResourceManager.NeonPostgres.OrganizationResource>> GetOrganizationResourceAsync(string organizationName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.ResourceManager.NeonPostgres.OrganizationResourceCollection GetOrganizationResources() { throw null; }
+    }
+    public partial class MockableNeonPostgresSubscriptionResource : Azure.ResourceManager.ArmResource
+    {
+        protected MockableNeonPostgresSubscriptionResource() { }
+        public virtual Azure.Pageable<Azure.ResourceManager.NeonPostgres.OrganizationResource> GetOrganizationResources(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.AsyncPageable<Azure.ResourceManager.NeonPostgres.OrganizationResource> GetOrganizationResourcesAsync(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+    }
+}
+namespace Azure.ResourceManager.NeonPostgres.Models
+{
+    public static partial class ArmNeonPostgresModelFactory
+    {
+        public static Azure.ResourceManager.NeonPostgres.Models.OrganizationProperties OrganizationProperties(Azure.ResourceManager.NeonPostgres.Models.MarketplaceDetails marketplaceDetails = null, Azure.ResourceManager.NeonPostgres.Models.UserDetails userDetails = null, Azure.ResourceManager.NeonPostgres.Models.CompanyDetails companyDetails = null, Azure.ResourceManager.NeonPostgres.Models.ResourceProvisioningState? provisioningState = default(Azure.ResourceManager.NeonPostgres.Models.ResourceProvisioningState?), Azure.ResourceManager.NeonPostgres.Models.PartnerOrganizationProperties partnerOrganizationProperties = null) { throw null; }
+        public static Azure.ResourceManager.NeonPostgres.OrganizationResourceData OrganizationResourceData(Azure.Core.ResourceIdentifier id = null, string name = null, Azure.Core.ResourceType resourceType = default(Azure.Core.ResourceType), Azure.ResourceManager.Models.SystemData systemData = null, System.Collections.Generic.IDictionary<string, string> tags = null, Azure.Core.AzureLocation location = default(Azure.Core.AzureLocation), Azure.ResourceManager.NeonPostgres.Models.OrganizationProperties properties = null) { throw null; }
+    }
+    public partial class CompanyDetails : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.NeonPostgres.Models.CompanyDetails>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.NeonPostgres.Models.CompanyDetails>
```

Comment:
The current name is too generic. Please rename it to something more specific and appropriate, such as `NeonCompanyDetails`.